strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/storage/checksum/ChecksumCacheManagerTest.java

```diff
@@ -7,15 +7,15 @@
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
 
-import org.junit.Ignore;
-import org.junit.Test;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
 
 /**
  * @author mtodorov
  */
-@Ignore
+@Disabled
```

Comment:
@fuss86,
(cc: @sbespalov)

Is this test also disabled in the `master` and your branch? If yes, we'll need to address it in a follow-up issue. (@fuss86, Could you please look into it as part of the work we were discussing?).